src/server/frontend_wayland/CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ set(
   window_wl_surface_role.cpp    window_wl_surface_role.h
   wl_surface.cpp                wl_surface.h
   wl_seat.cpp                   wl_seat.h
-  keyboard_helper.cpp           keyboard_helper.h
+  keyboard_helper.cpp
```

Comment:
I think removing this header from the project is fallout from not having a `KeyboardHelper` interface in shell